### PR TITLE
Add PACIFICA_2020 fingerprint

### PR DESCRIPTION
**Pull request by ErichMoraga**

### CODE REVIEW

**Comment by pd0wm** [comment]
You can't just sneak in fingerprinting 2.0 like that 😉

We should be a able to get much more information and more ECUs. Let's check what the manufacturers software does when requesting the FW versions.

**Comment by ErichMoraga** [comment]
> You can't just sneak in fingerprinting 2.0 like that 
> 
> We should be a able to get much more information and more ECUs. Let's check what the manufacturers software does when requesting the FW versions.

Are you suggesting this?
https://github.com/commaai/openpilot/wiki/Fingerprinting#fingerprinting-20-alternative

Anyway, I'm going to sneak FPv1 as a new subsection of the 2020 Pacifica too, just in case.  The 2021 Pacifica has many MANY differences from the 2020, but apparently works fine when FP'd as a 2020.

**Comment by jyoung8607** [comment]
> Are you suggesting this?
> https://github.com/commaai/openpilot/wiki/Fingerprinting#fingerprinting-20-alternative

The specific query types end up being important, and sometimes other details. That's why often we need to see what the manufacturer diagnostic software does. In the alternative (or in addition), somewhere I still have a modified version of Panda's [`query_fw_versions.py`](https://github.com/commaai/panda/blob/master/examples/query_fw_versions.py) that rolls through [every UDS-standard query on every address](https://github.com/commaai/openpilot/issues/1238#issuecomment-620142101) and can be modified to do manufacturer-specific queries as well.

All the picky little details would be a lot of typing that doesn't belong in this PR. If you have a population of cooperative Chrysler users and you're interested in tag-teaming an effort to get them FPv2, I can back you up on the diagnostic details, finding the right address and query set, and Chrysler specific FPv2 code. If so, let's take this over to a Github Discussion.

**Comment by ErichMoraga** [comment]
https://github.com/commaai/openpilot/discussions/21112

**Comment by pd0wm** [comment]
Can you only add the fingerprint for this car? This PR changes many lines now.

**Comment by ErichMoraga** [comment]
> Can you only add the fingerprint for this car? This PR changes many lines now.

I basically fixed the FPv1 formatting, making it consistent throughout the file.

**Comment by n432** [comment]
I installed https://smiskol.com/fork/erichmoraga/patch-1, and tested that it works in the following video:
e26a05d431cc1fbf|2021-06-09--06-32-58--0
https://my.comma.ai/e26a05d431cc1fbf/1623234777889/1623235381659

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -32,49 +32,61 @@ class CAR:
...
+  }],
 }
 
 
+STEER_THRESHOLD = 100
```
Comment (adeebshihadeh) [inline selfdrive/car/chrysler/values.py:88]: where did this come from?

File: selfdrive/car/chrysler/values.py [hunk 2]
```diff
@@ -32,46 +32,56 @@ class CAR:
...
..., 464: 8, 469: 8, 480: 8, 500: 8, 501: 8, 512: 8, 514: 8, 516: 7, 517: 7, 520: 8, 524: 8, 526: 6, 528: 8, 532: 8, 536: 8, 542: 8, 544: 8, 557: 8, 559: 8, 560: 8, 564: 8, 571: 3, 579: 8, 584: 8, 608: 8, 624: 8, 625: 8, 632: 8, 639: 8, 650: 8, 656: 4, 658: 6, 660: 8, 669: 3, 671: 8, 672: 8, 676: 8, 678: 8, 680: 8, 683: 8, 703: 8, 705: 8, 706: 8, 709: 8, 710: 8, 711: 8, 719: 8, 720: 6, 729: 5, 736: 8, 746: 5, 752: 2, 754: 8, 760: 8, 764: 8, 766: 8, 770: 8, 773: 8, 776: 8, 779: 8, 782: 8, 784: 8, 792: 8, 793: 8, 794: 8, 795: 8, 799: 8, 800: 8, 801: 8, 802: 8, 803: 8, 804: 8, 808: 8, 816: 8, 817: 8, 820: 8, 825: 2, 826: 8, 832: 8, 838: 2, 848: 8, 853: 8, 856: 4, 860: 6, 863: 8, 882: 8, 886: 8, 897: 8, 906: 8, 924: 8, 926: 3, 937: 8, 938: 8, 939: 8, 940: 8, 941: 8, 942: 8, 943: 8, 947: 8, 948: 8, 962: 8, 969: 4, 973: 8, 974: 5, 979: 8, 980: 8, 981: 8, 982: 8, 983: 8, 984: 8, 992: 8, 993: 7, 995: 8, 996: 8, 1000: 8, 1001: 8, 1002: 8, 1003: 8, 1008: 8, 1009: 8, 1010: 8, 1011: 8, 1012: 8, 1013: 8, 1014: 8, 1015: 8, 1024: 8, 1025: 8, 1026: 8, 1031: 8, 1033: 8, 1050: 8, 1059: 8, 1098: 8, 1100: 8, 1216: 8, 1250: 8, 1543: 8, 1856: 8, 1890: 8, 1898: 8, 2015: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8
```
Comment (adeebshihadeh) [inline selfdrive/car/chrysler/values.py:49]: merge this with the existing fingerprint

**Review by ErichMoraga — COMMENTED** [review]
Looks good

**Comment by adeebshihadeh** [comment]
This is good to merge once the comment is addressed.

File: selfdrive/car/chrysler/values.py (hunk 2 above)
Comment (ErichMoraga) [reply selfdrive/car/chrysler/values.py]: OK, I combined 2021 Pacifica FPv1 into existing FPv1

**Pull request merged**